validator/remote/src/test/java/tech/pegasys/teku/validator/remote/sentry/SentryNodesConfigLoaderTest.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright ConsenSys Software Inc., 2022
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.validator.remote.sentry;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import java.io.IOException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+@SuppressWarnings("OptionalGetWithoutIsPresent")
+class SentryNodesConfigLoaderTest {
+
+  public static final String SENTRY_NODE_FULL_CONFIG_JSON =
+      "src/test/resources" + "/sentry_node_full_config.json";
+  public static final String SENTRY_NODE_MINIMUM_CONFIG_JSON =
+      "src/test/resources" + "/sentry_node_minimum_config.json";
+  public static final String SENTRY_NODE_MISSING_ENDPOINTS_CONFIG_JSON =
+      "src/test/resources" + "/sentry_node_missing_endpoints_config.json";
+  public static final String SENTRY_NODE_MISSING_DUTIES_PROVIDER_CONFIG_JSON =
+      "src/test" + "/resources/sentry_node_missing_duties_provider_config.json";
+  public static final String SENTRY_NODE_MISSING_BLOCK_HANDLER_CONFIG_JSON =
+      "src/test/resources" + "/sentry_node_missing_block_handler_config.json";
+  public static final String SENTRY_NODE_MISSING_ATTESTATION_PUBLISHER_CONFIG_JSON =
+      "src/test" + "/resources/sentry_node_missing_attestation_publisher_config.json";
```

Comment:
there's an example of reading resources at `GetStateTest` in the `shouldHandle200`, using `Resources.getResource`
It might be cleaner? I'm not fussed either way particularly, but it's a fairly neat way to read resources and means its in the same route as your testing class etc...